lexic.txt

```diff
@@ -1,37 +0,0 @@
-Alphabet:
-
-a. Upper (A-Z) and lower case letters (a-z) of the English alphabet
-b. Underline character '_';
-c. Decimal digits (0-9);
-
--operators + - * / < <= = >= eq mod ** not
-
-- separators [ ] { } ; space
-
-- reserved words: init verify otherwise escape true untrue while for func in
-
-integer:
-
-<int>::= 0 | <nonzero>
-<nonzero>::=<nonzerodigit>|-<nonzerodigit>|<nonzerodigit><digitseq>|-<nonzerodigit><digitseq>
-<digitseq>::= <digit>|<digit><digitseq>
-<digit>::=0|<nonzerodigit>
-<nonzerodigit>::=1...
-
-
- <letter> ::= "a" | "b" | ... | "Z"
```

Comment:
I like the fact that you formatted this one better but still not very intuitive that you go form 'a' to 'z' and than from 'A' to 'Z'
"a" | "b" | ... | "z" | "A" | ... | "Z"